pkg/operator/csi/csicontrollerset/csi_controller_set.go

```diff
@@ -95,11 +96,13 @@ func (c *CSIControllerSet) WithConditionalStaticResourcesController(
 	manifests resourceapply.AssetFunc,
 	files []string,
 	shouldCreateFnArg, shouldDeleteFnArg resourceapply.ConditionalFunction,
+	manifestObjModifier func(runtime.Object) (runtime.Object, error),
 ) *CSIControllerSet {
 	c.conditionalStaticResourcesControllers = append(c.conditionalStaticResourcesControllers, staticresourcecontroller.NewStaticResourceController(
 		name,
 		manifests,
-		[]string{},
```

Comment:
Why do we change this to nil?. Accidental change?.

Reply:
since I was changing the line below, I also changed this line to improve readability